code/modules/client/preferences/migrations/tts_blip_migration.dm

```diff
@@ -6,7 +6,10 @@
 	var/sound_blips_enabled = savefile.get_entry("sound_tts_blips")
 	if(sound_blips_enabled)
 		write_preference(GLOB.preference_entries[/datum/preference/choiced/sound_tts], TTS_SOUND_BLIPS)
+		savefile.remove_entry("sound_tts_blips")
+		savefile.save()
```

Comment:
What is this and why do we need it? The migration has already been performed on everyone

Reply:
You should always expect to have dead preferences in the prefs file